doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,281 @@
+This is due to the way the underlying HTTP library (libmicrohttpd) works.
+	
+    Content-Length: 34	
+
+[Top](#top)
+ 
+## 3. Content-Type 
+
+It is used to specify the MIME type of the request or response. Typically its value is `application/json` (for JSON MIME type) although some operations in the NGSIv2 API also allow `text/plain`.
+
+    curl ... -H 'Content-Type: application/json'
+
+This request will send data in well defined JSON format.
+
+[Top](#top)
+ 
+## 4. Connection
+ 
+ This header is used by both requests and responses.
+ Orion is not currently doing anything with this header (apart from checking if its value is `close` to print a log trace).
+	
+    Connection: Keep-Alive
+    	
+[Top](#top)	
+  
+## 5. Expect
+
+Expect HTTP header is used in incoming HTTP requests received by Orion.
+It is used when sending a large request body, on response Orion sends only `Expect:` i.e. blank, if it is able to handle the request. For more details, follow general documentation of Expect HTTP header [Link1](https://flaviocopes.com/http-request-headers/), [Link2](https://tools.ietf.org/html/rfc7231#section-5.1.1) or [Link3](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields).
+
+
+    Expect:
+	
+[Top](#top)	
+	
+## 6. Host
+
+This HTTP header is used in Orion responses.
+Orion includes this header in outgoing request (notifications and forwarded updates/queries).This header provides the port number of notification receiver. 	
+	
+    Host: localhost:1028
+		
+[Top](#top)
+
+## 7. Origin
+
+Origin is used in incoming HTTP requests received by Orion.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md).
+	
+    Origin: http://www.example-social-network.com
+		
+[Top](#top)	
+  
+## 8. User-Agent
+
+User-Agent HTTP header is used in notifications and forwarded requests.
+It describes which version of Orion we are using along with its transfer library.
+It gives the string that identifies the user agent.
+
+    User-Agent: orion/2.2.0-next libcurl/7.29.0     (in notifications and forwarded requests)
+
+[Top](#top)		
+		
+## 9. X-Forwarded-For
+
+It is used in incoming HTTP requests received by Orion.
+X-Forwarded-For header overrides the original IP of the HTTP request as source of the transaction.
+X-Real-IP and X-Forwarded-For (used by a potential proxy on top of Orion) overrides IP.
+X-Real-IP takes preference over X-Forwarded-For, if both appear.
+	
+    X-Forwarded-For: 129.78.138.66
+	
+[Top](#top) 
+	
+## 10. X-Real-IP
+
+X-Forwarded-For header overrides the original IP of the HTTP request as source of the transaction.
+X-Real-IP and X-Forwarded-For (used by a potential proxy on top of Orion) overrides IP.
+X-Real-IP takes preference over X-Forwarded-For, if both appear.
```

Comment:
Maybe both sections could be merged (i.e. `## N. X-Fowarded-For and X-Real-IP`).

Reply:
Fixed in 512f9cd